toolkit/phone-assessment-interviews/personal-relationships.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: article
+permalink: /toolkit/phone-assessment-interviews/personal-relationships/
+section: hiring-phases
+sidenav: hiring-phases
+phase: phone-assessment-interviews
+title: Guidance on Personal Relationships with Applicants
+description: Use this guidance to know whether the SME should recuse themselves from assessing an applicant.
+---
+
+It's common for SMEs participating in a hiring action to know applicants. SMEs may interview people they know, supervise, and work with, such as contract professionals or fellow public servants, except in these cases:
+
+- The SME has a personal relationship with the applicant outside of work, whether or not the relationship is known to others.
+- There is any _appearance_ of bias from the SME toward the applicant, such as when it's well known that the SME and applicant have regular disagreements, or that the SME and the applicant have a unique bond.
+
+In general, if the SME and the applicant have occasionally attended the same office happy hour, there is no conflict. However, if the SME and applicant are regular drinking buddies, the SME should recuse themselves.
```

Comment:
i think we should expand the drinking buddies and say "regular drinking buddies or known to go out to dinner from time to time"

Reply:
It is weird that "drinking buddies" feels so gendered to me? Maybe just:

> If the SME and applicant regularly socialize together outside of work, the SME should recuse themselves.

Reply:
Good call, thanks, Martha.